calico/reference/resources/globalnetworkpolicy.mdx

```diff
@@ -140,6 +142,18 @@ for how `doNotTrack` and `preDNAT` and `applyOnForward` can be useful for host e
 
 <Servicematch />
 
+### Performance Hints
+
+Performance hints provide a way to tell {{prodname}} about the intended use of the policy so that it may
```

Comment:
by "may" do you mean "it might possibly", or should this be "will" or "can" process it more efficiently?

Reply:
Yes, "it might possibly".  The hint may help or not, we'll do our best!